srcpkgs/MultiMC/files/MultiMC.sh

```diff
@@ -1,2 +0,0 @@
-#!/bin/sh
-/usr/libexec/multimc/MultiMC -d ~/.multimc
```

Comment:
Can't you change it to `/usr/bin/multimc -d ~/.multimc`, so we don't break the old behavior. (assuming that is the new path of the executable)

Reply:
I could but I'm not sure about having both multimc and MultiMC in /usr/bin/, let me check if I can get away with moving multimc to /usr/libexec/multimc/.